src/modules/Popup/Popup.d.ts

```diff
@@ -109,6 +109,9 @@ export interface StrictPopupProps extends StrictPortalProps {
 
   /** Element to be rendered within the confines of the viewport whenever possible. */
   keepInViewPort?: boolean
+
+  /** Disables the popup and only renders the element passed via trigger prop */
```

Comment:
Suggestion:

```suggestion
  /** A disabled popup only renders its trigger. */
```

This will also be the title of the doc site example.

The `disabled` prop is a new popup state, so we should add a doc site example to `docs/src/examples/modules/Popup/States/PopupExampleDisabled.js`.

Our docs are powered by the structure of the examples directory.  You can reference `/State` examples in other components for a guide as to how to structure this one for the Popup.

Reply:
Thank you for the feedback!

I've renamed the description as per the suggestion and also added the State example.
Let me know what else can be improved.